.github/workflows/docs.yml

```diff
@@ -44,6 +44,7 @@ jobs:
           ref: ${{ env.COMMIT }}
           repository: ${{ env.PR_REPO }}
           fetch-depth: 0
+          submodules: true
```

Comment:
There are 2 checkout commands in this file

Reply:
Fixed :  e721e3d